content/en/docs/ops/deployment/security-model/index.md

```diff
@@ -0,0 +1,218 @@
+Note that this automatic detection relies on Istio associating the traffic to a Service.
+[Unsupported traffic types](/docs/ops/configuration/traffic-management/traffic-routing/#unmatched-traffic) or [configuration scoping](/docs/ops/configuration/mesh/configuration-scoping/) can prevent this.
+
+When [connecting to a backend](/docs/concepts/security/#secure-naming), the set of allowed identities is computed, at the Service level, based on the union of all backend's identities.
+{{< /tab >}}
+
+{{< tab name="Ambient mode" category-value="ambient" >}}
+In ambient mode, Istio will automatically use mTLS when connecting to any backend that supports mTLS.
+
+When [connecting to a backend](/docs/concepts/security/#secure-naming), the expected identity of the specific workload is verified to match.
+
+These properties differ from sidecar mode in that they are properties of individual workloads, rather than of the service.
+This enables more fine-grained authentication checks, as well as supporting a wider variety of workloads.
+{{< /tab >}}
+{{< /tabset >}}
+
+### Server mTLS
+
+By default, Istio will accept mTLS and non-mTLS traffic (often called "permissive mode").
+Users can opt-in to strict enforcement by writing `PeerAuthentication` or `AuthorizationPolicy` rules requiring mTLS.
+
+When mTLS connections are established, the peer certificate is verified.
+Additionally, the peer identity is verified to be within the same trust domain.
+To verify only specific identities are allowed, an `AuthorizationPolicy` can be used.
+
+## Compromise types explored
+
+Based on the above overview, we will consider the impact on the cluster if various parts of the system are compromised.
+In the real world, there are a variety of different variables around any security attack:
+
+* How easy it is to execute
+* What prior privileges are required
+* How often it can exploited
+* What the impact is (total remote execution, denial of service, etc).
+
+In this document, we will primarily consider the worst case scenario: a compromised component means an attacker has complete remote code execution capabilities.
+
+### Workload compromise
+
+In this scenario, an application workload (Pod) is compromised.
+
+A pod [*may* have access](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#opt-out-of-api-credential-automounting) to its service account token.
+If so, a workload compromise can move laterally from a single Pod to compromising the entire service account.
+
+{{< tabset category-name="dataplane" >}}
+{{< tab name="Sidecar mode" category-value="sidecar" >}}
+In the sidecar model, the proxy is co-located with the pod, and runs within the same trust boundary.
+A compromised application can tamper with the proxy through the admin API or other surfaces.
+It should be assumed that a compromised workload also includes a compromise of the sidecar proxy.
```

Comment:
Should point out that this includes exfiltration of the private key allowing another malicious agent to impersonate the workload for egress.